lib/laforge/activerecord.rb

```diff
@@ -61,6 +61,8 @@ def forge_attributes(attributes: nil, sources: nil)
       forged_attrs = {}
 
       filter_loaded_data_entries(attributes: attributes, sources: sources, present: true).sort_by(&:priority_with_fallback).reverse.uniq(&:attribute_name).each do |data_entry|
+        next unless respond_to?("#{data_entry.attribute_name}=")
```

Comment:
For the interested reader, here's where Rails does its check https://github.com/rails/rails/blob/5-2-stable/activemodel/lib/active_model/attribute_assignment.rb#L50.